tidb-cloud/tidb-cloud-release-notes.md

```diff
@@ -8,6 +8,53 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.
 
+## April 25, 2023
+
+**General changes**
+
+- For the first five Serverless Tier clusters in your organization, TiDB Cloud provides a free usage quota for each of them as follows:
+
+    - Row storage: 5 GiB
+    - [Request Units](/tidb-cloud/tidb-cloud-glossary.md#request-unit) (RUs): 50 million RUs per month
```

Comment:
It seems that this anchor link does not work well.

Reply:
It is is a new term to be added in https://github.com/pingcap/docs/pull/13335/files.